crates/biome_css_formatter/src/css/properties/generic_property.rs

```diff
@@ -12,6 +12,45 @@ impl FormatNodeRule<CssGenericProperty> for FormatCssGenericProperty {
             value,
         } = node.as_fields();
 
+        let is_dash_identity = name.clone().unwrap().as_css_dashed_identifier().is_some();
+
+        let has_comma = value.iter().any(|v| v.text().eq(","));
```

Comment:
Checking for a comma (`,`) like this isn't correct. You should rely on the CST/AST, so you don't need to transform a node into a test.

You can use the playground to surgically understand how to query it: https://biomejs.dev/playground/?lintRules=all&files.main.css=aAB0AG0AbAAgAHsACgAJAGYAbwBuAHQALQBmAGEAbQBpAGwAeQA6ACAAcwB5AHMAdABlAG0ALQB1AGkALAAgAC0AYQBwAHAAbABlAC0AcwB5AHMAdABlAG0AOwAKAH0A

(Click on the comma on the left side, and it should point to the relative node on the right, in the syntax tab)